src/rdbms/mongoose_rdbms_odbc.erl

```diff
@@ -76,18 +78,20 @@ query(Connection, Query, Timeout) ->
               Fields :: [binary()], Statement :: iodata()) ->
                      {ok, {binary(), [fun((term()) -> tuple())]}}.
 prepare(Connection, Name, Table, Fields, Statement) ->
-    try prepare2(Connection, Table, Fields, Statement)
+    TabCols = fields_to_tabcol(Fields, Table),
```

Comment:
Just to make sure: the only reason we don't just accept `TabCols` as an argument instead of `Fields` is that you consider the `'tab.col'` syntax better than `{tab, col}`, right?

Reply:
the main reason is that it's the same syntax as in SQL queries. 
We don't do "SELECT {tab, col}".

Reply:
Also, it's the same reason why Fields are atoms, but we convert it to binaries in the prepare call.
It just looks a bit cleaner.